applications/StructuralMechanicsApplication/custom_strategies/custom_convergencecriterias/error_mesh_criteria.h

```diff
@@ -68,25 +68,29 @@ class ErrorMeshCriteria
 
     KRATOS_CLASS_POINTER_DEFINITION( ErrorMeshCriteria );
 
+    /// The definition of the base ConvergenceCriteria
     typedef ConvergenceCriteria< TSparseSpace, TDenseSpace >        BaseType;
 
-    typedef TSparseSpace                                     SparseSpaceType;
+    /// The definition of the current class
+    typedef ErrorMeshCriteria< TSparseSpace, TDenseSpace >         ClassType;
 
-    typedef typename BaseType::TDataType                           TDataType;
+    /// The data type
+    typedef typename BaseType::TDataType TDataType;
 
-    typedef typename BaseType::DofsArrayType                   DofsArrayType;
+    /// The dofs array type
+    typedef typename BaseType::DofsArrayType DofsArrayType;
 
-    typedef typename BaseType::TSystemMatrixType           TSystemMatrixType;
+    /// The sparse matrix type
+    typedef typename BaseType::TSystemMatrixType TSystemMatrixType;
 
-    typedef typename BaseType::TSystemVectorType           TSystemVectorType;
+    /// The dense vector type
+    typedef typename BaseType::TSystemVectorType TSystemVectorType;
```

Comment:
absolutely minor ... but ... why the T in the name?

Reply:
The T was already there, i am changing spaces